www/app/Http/Controllers/CodexAuthController.php

```
@@ -217,65 +239,163 @@ protected function isValidCredentialFile(array $data): bool
-                'enabled' => true,
-                'is_default' => true,
-                'inherit_workspace_tools' => true,
+        // Store initial "starting" state so the polling endpoint can respond immediately
+        Cache::put('codex_device_auth', [
+            'status' => 'starting',
+            'started_at' => time(),
+            'expires_at' => time() + 960,
+        ], 960);
+
+        // Dispatch the long-running job to the queue container
+        \App\Jobs\StartCodexDeviceAuthJob::dispatch();
```

PHPStan (2.1.54)

[error] Call to an undefined static method App\Jobs\StartCodexDeviceAuthJob::dispatch().


(staticMethod.notFound)